src/Cohesity.Powershell/Scripts/Routes/Get-CohesityRoutes.ps1

```diff
@@ -0,0 +1,78 @@
+function Get-CohesityRoutes {
+    <#
+        .SYNOPSIS
+        Get the routes.
+        .DESCRIPTION
+        The Get-CohesityRoutes function is used to get routes.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Get-CohesityRoutes
+        .EXAMPLE
+        Get-CohesityRoutes -FilterName INTERFACE-GROUP-NAME -FilterValue "intf_group1"
+        .EXAMPLE
+        Get-CohesityRoutes -FilterName DESTINATION-NETWORK -FilterValue "1.2.4.14/32"
+    #>
+    [CmdletBinding(DefaultParameterSetName = 'Default')]
```

Comment:
Where this 'DefaultParameterSetName' play a role in this cmdlet?

Reply:
We need to use the cmdlet with FilterName & FilterValue or none should be provided in the command line, ParameterSetName helps in organizing this behavior.

Reply:
Nice use fo ParameterSetName, so the mandatory applies only to the filter set ? Btw what if someone uses : 
Get-CohesityRoutes -FilterName DESTINATION-NETWORK -FilterValue "1.2.4.14/32"

Reply:
ParameterSetName has many usage and we are yet to exploit all of them. However, ParameterSetName is the keyword for grouping parameters and as we keep writing we will keep improving the usage of ParameterSetName and write more and more complex cmdlets. 
Between, Get-CohesityRoutes -FilterName DESTINATION-NETWORK -FilterValue "1.2.4.14/32", would filter out and display the details of "1.2.4.14/32".